Config/tile.py

```diff
@@ -6,4 +6,74 @@ def __init__(self,N=0,E=0,S=0,W=0):
 		self.N = N
 		self.E = E
 		self.S = S
-		self.W = W
\ No newline at end of file
+		self.W = W
+
+
+	def getIndex(self) -> int:
+		"""
+		Returns the tile number for current tile.
+		:return: 0 if the tile is empty, 1 to 10 according to convention given in editeurConfig.py if tile matches,
+		11 by default for all other cases.
+		"""
+		s = "{}{}{}{}".format(self.N, self.E, self.S, self.W)
+		if s == "0000":
```

Comment:
switch... case statements (match... case actually) have been introduced in Python 3.10, so that's an option. But I'm with if... else if we want to keep backward compatibility.

Reply:
I agree with the commentary. For the backward compatibility I think it would be better to keep it this way.

The format in the code snippet is corrected in commit c459d849aec6f0e6e979d0acd679c2dcc7114783